rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -564,8 +564,7 @@ def test_exec(config, ssh_con):
                 raise TestExecError("Failed to enable user")
         if config.delete_user:
             user_id = each_user["user_id"]
-            cmd = f"radosgw-admin user rm --uid='{user_id}'"
-            out = utils.exec_shell_cmd(cmd)
+            out = reusable.remove_user(each_user)
```

Comment:
Could you please check scripts available in our repos and change it with `reusable.remove_user(each_user)` where we are deleting user using cli

Reply:
done uday